src/elasticsearch/addl_index_transformations/portal/__init__.py

```diff
@@ -40,6 +40,16 @@ def _get_version():
     return 'no-build-file'
 
 
+def get_config():
```

Comment:
Where it matters, we're just reading the yaml rather than using this method. If just referencing this file is fine, can get rid of this method.